src/ol_infrastructure/infrastructure/aws/network/__main__.py

```diff
@@ -38,6 +36,19 @@ def vpc_exports(vpc: OLVPC) -> Dict:
         'Name': f'{stack_name} Data Services'})
 data_vpc = OLVPC(data_vpc_config)
 
+residential_mitx_vpc_config = OLVPCConfig(
+    vpc_name=f'mitx-{env_suffix}',
+    cidr_block=get_config('residential_vpc:cidr_block'),
+    num_subnets=4,
```

Comment:
Guess it would be helpful to either add a comment here or in the Readme explaining why 4 subnets and 3 elsewhere

Reply:
Will do